app/queues/dns/dns-flow.server.ts

```diff
@@ -1,23 +1,24 @@
 import { FlowProducer } from 'bullmq';
 import {
-  addDbRecordQueueName,
-  createDnsRecordQueueName,
+  dbRecordUpdateQueueName,
+  dnsRecordUpdateQueueName,
   checkDnsStatusQueueName,
   syncDbStatusQueueName,
-} from './create-record-worker.server';
+} from './dns-record-worker.server';
 import { redis } from '~/lib/redis.server';
 
 import type { FlowJob } from 'bullmq';
-import type { JobRecord } from './create-record-worker.server';
+import type { JobRecord } from './dns-record-worker.server';
+import { WorkType } from './dns-record-worker.server';
 
 const flowProducer = new FlowProducer({ connection: redis });
 
 export const addDnsRequest = async ({ username, type, name, value }: JobRecord) => {
   // Step 1. Create a record in MySQL for a domain with pending status
   const addDbRecord: FlowJob = {
```

Comment:
Question: should we be adding the initial record to the db *before* we put this job in the queue?  I ask because the front-end is going to want to be able to query for this record immediately, and poll on it to see when it's completed (i.e., when `status` changes).

In other words, should we put the record in the db with `pending` status, then add the job to the queue and return?

Reply:
Totally makes sense. It should be working as you stated. I will think about how it should be done.